webhooks/github_event_handler.py

```diff
@@ -42,6 +42,7 @@ def get_metadata(event_type, payload):
                 'owner': payload['repository']['owner'].get('name'),
                 'repo': payload['repository']['full_name'],
                 'branch': payload['ref'].replace('refs/heads/', ''),
+                'target_branch': payload['ref'].replace('refs/heads/', ''),
```

Comment:
Actually "target_branch" is not a meaningful attribute of a "push" event but needs to be somehow defined so the code later on can be simpler. Does it make sense?